.env

```diff
@@ -21,6 +21,19 @@ POSTGRES_PASSWORD=postgres
 HASH_KRATOS_PG_USER=kratos
 HASH_KRATOS_PG_PASSWORD=kratos
 HASH_KRATOS_PG_DATABASE=kratos
+ORY_KRATOS_PUBLIC_URL=http://127.0.0.1:4433
+ORY_KRATOS_ADMIN_URL=HTTP://127.0.0.1:4434
+KRATOS_API_KEY=secret
```

Comment:
Ideally we'd make these naming patterns consistent at some point. I moved them into the same place to at least keep track of the Kratos environment variables more easily.